### ISSUE TITLE
Alt Gr problem in chat.

### ISSUE THREAD

**Issue by norgor** [notsecure/uTox#902]
Pressing 'Alt Gr + A' selects the whole text, pressing 'Alt Gr + Z' works like backspace.
The real issue comes in when you try to use another keyboard layout like Polish, where to write some of the letters you need to use these key combinations. When you try to type the letter 'ą' you need to use the first combination 'Alt Gr + A' which deletes the whole line because it selects the whole input and then tries to append the character.
:)

**Comment by tsudoko** [comment]
Can't reproduce, selecting/undoing works for me with ctrl-a/ctrl-z, not altgr-a/altgr-z.

**Comment by GrayHatter** [comment]
@norgor can you provide more information on this one?

**Comment by norgor** [comment]
I just found out that it doesn't affect the English keyboard layout, but others like Norwegian and Polish are affected.

**Comment by tsudoko** [comment]
It's fine here on both Polish and Norwegian layouts. Are you using Windows?

**Comment by norgor** [comment]
Yes, I am using Windows 8.1. I asked some of my friends to change their keyboard layout, they had the same issue.

**Comment by tsudoko** [comment]
It's a Windows-only issue, then; doesn't happen either on Wine or Gentoo.

**Comment by ElLamparto** [comment]
I confirm the problem. There is a number of Polish characters which are created by pressing <ALT GR> and then a key. For example <ALT GR><l> gives 'ł'. Other characters are for example: ą ę ń ż ó.
µTox process those letters as CONTROL CHARACTERS and that's the problem!
'ł' acts like backspace, while another one (I don't remeber which), WIPES ALL the written text from the edit field.

Therefore it is a critical bug.

Imagine someone who types slowly (my mother :) and then by pressing a key, looses the whole message before sending!

**Comment by ElLamparto** [comment]
One more thing. This bug appears on WinXP 32bit. On Win7 64 bit there is no problem.

**Comment by mlazowik** [comment]
OS X Maverics, the same. Big letters work (both with shift and caps-lock). Cannot enter small ones, but also they don't seem to be working as shortcuts as others have reported.

**Comment by turbogiovanni** [comment]
This bug is still persistent - can we get some feedback on it? Win 7 x64 sp1 system language is set to Polish and keyboard layout set to Polish (Programmers). Is it possible to disable keyboard shortucs for uTox anyway? This could be a good workaround.

**Comment by GrayHatter** [comment]
what version are you on?

**Comment by turbogiovanni** [comment]
64bit 0.15.0 my friend.
